### PR TITLE
feat: Added gas price fee history

### PR DESCRIPTION
**Pull request by DarkSorrow**
notion.so/STAT-GAS-price-chart-ef47c4d23f9842cca0ce023f2f9abbc8

### CODE REVIEW

File: src/app/dialogs/gas-dialog/gas-dialog.component.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+              private log: NGXLogger) {
+                super();
+  }
+
+  ngAfterViewInit(): void {
+    this.loadData();
+  }
+
+  private loadData(): void {
+    this.httpService.getLastTvls().subscribe(data => {
```
Comment (belbix) [inline src/app/dialogs/gas-dialog/gas-dialog.component.ts:31]: please, use getHarvestTxHistoryByRange

File: src/app/dialogs/gas-dialog/gas-dialog.component.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<div *ngIf="ready === true; then readyBlock else loadingBlock" id="ready"></div>
+
+<ng-template #loadingBlock>
+  <app-loading-spinner></app-loading-spinner>
+</ng-template>
+
+<ng-template #readyBlock>
+  <div class="dialog-card">
+    <div class="title">
+      {{data.title}}
```
Comment (belbix) [inline src/app/dialogs/gas-dialog/gas-dialog.component.html:10]: use static title

Comment (DarkSorrow) [reply src/app/dialogs/gas-dialog/gas-dialog.component.html]: Changed to hard coded title

File: src/app/dialogs/gas-dialog/gas-dialog.component.ts (hunk 1 above)
Comment (DarkSorrow) [reply src/app/dialogs/gas-dialog/gas-dialog.component.ts]: Replaced with a new static value for month

**Pull request merged**